api/src/main/java/com/qaengine/security/SecurityConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.qaengine.security;
+
+public class SecurityConstants {
+    public static final String SECRET = "SecretKeyToGenJWTs";
+    public static final long EXPIRATION_TIME = 864_000_000_000L; // 10,000 days
```

Comment:
Just a comment. 
In reality jwt tokens should last max 24h,
Normal would be 15min soft expiration and 30 min until hard expiration. So during 15-30 mins you should be able to renew the token, which needs a separate endpoint.

Reply:
I was thinking it might be annoying to develop when you have to renew the token every time, especially in Postman. Maybe switch the expiration time in the third iteration?

Reply:
Also I just noticed I forgot to change the JWT secret. *facepalm*

Reply:
Yep, authentication was initially planned in third iteration so good for now.